pages/docs/tutorials/streetlights.md

```diff
@@ -15,10 +15,21 @@ You want to create a system capable of turning on/off the streetlights depending
 - Based off the events, your application can make a decision regarding turning the streetlight off.
 - Your application is not aware of how many streetlights are publishing events - it just connects to the broker and receives all events
 
-# Technology
+# Installation Guide
 
 You'll use Node.js to code the APIs and Mosquitto as the message broker. The selected technology is irrelevant here, since everything explained in this tutorial is applicable to any other programming language and message brokers.
 
```

Comment:
@luphieanza Let's keep the wording in the present tense because the idea is to guide them through it.

If we use past tense verbs/words, then it implies the reader already did the work.